packages/typescript/src/components/Interface.tsx

```diff
@@ -60,6 +69,10 @@ export function InterfaceMember(props: InterfaceMemberProps) {
   } else {
     return (
       <>
+        <Show when={Boolean(props.doc)}>
+          <JSDoc children={props.doc} />
```

Comment:
Since you're just passing as children, you could make `doc` children as well, instead of just string. Or at least take an array of strings which will allow passing multiple paragraphs.